Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM opensuse/amd64:42.3
+
+LABEL Maintainer="Vicente Zepeda <vzepedamas@suse.com>"
+
+
+# We have to use out-of-tree repos because several packages haven't been merged
+# into openSUSE:Factory yet.
+RUN zypper ar -f -p 10 -g obs://Virtualization:containers obs-vc && \
+	zypper --gpg-auto-import-keys -n ref && \
+	zypper -n up
+RUN zypper -n in \
+		bats \
+		git \
+		go \
+		golang-github-cpuguy83-go-md2man \
+		go-mtree \
+		jq \
+		libcap-progs \
+		make \
+		moreutils \
+		oci-image-tools \
+		oci-runtime-tools \
+		python-setuptools python-xattr attr \
+		skopeo \
+		tar
+
+ENV GOPATH /go
+ENV PATH $GOPATH/bin:$PATH
+RUN go get -u github.com/golang/dep/cmd/dep
+RUN go get -u github.com/golang/lint/golint
+RUN go get -u github.com/vbatts/git-validation && type git-validation
```

Comment:
Nitpicking: you could merge these three RUNs statements into a single one :wink:

Reply:
Done :+1: